src/core/Akka.Persistence/persistence.conf

```diff
@@ -71,11 +71,12 @@ akka.persistence {
         # `plugin-dispatcher` field.
         default-plugin-dispatcher {
             type = PinnedDispatcher
-            executor = "thread-pool-executor"
+            executor = "fork-join-executor"
```

Comment:
Pinned dispatchers use fork-join dispatchers in Akka.NET, which is the reverse from JVM Akka. Reason is that the fork-join dispatcher vs. threadpool dispatcher mean different things on the JVM versus Akka.NET. Our ForkJoinDispatcher is really a dedicated threadpool.